Add feedback*.svg that show Feedback scenarious.

johado opened the pull request:
Change-Id: Id42e61714ebc6ead2a69a93b716237701175213c

HansBusch commented:
Interesting idea to use graphviz.
Two remarks: 
1. Where is the change to the specification?
2. Not sure whether we should include source and makefiles in same folder.

johado commented:
Have now added added it to AccessControl Overview - not sure if that is the best place or not.

johado commented:
Now reworked the flow chart using a uml description and another web based tool after feedback from WG PACS telco.
Moved it to its own Use case examples appendix.

HansBusch commented:
TC yesterday discussed the proposal. Members liked the idea to put the source code close to the svg image. However we would like to see the source in a sub-folder called "source" in order to be able to automatically strip sources before uploading to the public website.

Please move the source uml file to folder doc/media/AccessControl/source.

johado commented:
flow.uml now moved to source directory and README.txt updated. Should README.txt also be moved to source dir?

HansBusch commented:
Last minor change request: can you please remove the readme. With the new structure it should be obvious. If you want to keep it please move it to the source folder.

johado commented:
README.tx moved to source dir. (Think it is useful)
